CondCore/JetMETPlugins/plugins/JetResolution_PayloadInspector.cc

```diff
@@ -49,9 +49,32 @@ namespace JME {
   public:
     JetResolutionVsEta()
         : cond::payloadInspector::Histogram1D<JetResolutionObject, SINGLE_IOV>(
-              "Jet Resolution", "#eta", NBIN_ETA, MIN_ETA, MAX_ETA, "Resolution") {}
+              "Jet Resolution", "#eta", NBIN_ETA, MIN_ETA, MAX_ETA, "Resolution") {
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Pt");
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Eta");
+      cond::payloadInspector::PlotBase::addInputParam("Jet_Rho");
+    }
 
     bool fill() override {
+      double par_Pt = 100.;
+      double par_Eta = 1.;
+      double par_Rho = 20.;
+
+      // Default values will be used if no input parameters
+      auto paramValues = cond::payloadInspector::PlotBase::inputParamValues();
+      auto ip = paramValues.find("Jet_Pt");
+      if (ip != paramValues.end()) {
+        par_Pt = std::stod(ip->second);
+      }
+      ip = paramValues.find("Jet_Eta");
+      if (ip != paramValues.end()) {
+        par_Eta = std::stod(ip->second);
```

Comment:
This `par_Eta` is never used (overwritten at [L96](https://github.com/cms-sw/cmssw/pull/41529/files#diff-2f814a0d8ef5675be3b3246f3f0b746b66a336a096f3c2177e051af3193dfe29R96)): please check

Reply:
It's the x-axis of the histogram. In L97 payload record is checked to be valid in the bin-range for the loop (L86). And you are right, input parameter Jet_Eta is not used.